rules/framework.bzl

```diff
@@ -1053,23 +1073,23 @@ apple_framework_packaging = rule(
         ),
         "deps": attr.label_list(
             mandatory = True,
-            cfg = apple_common.multi_arch_split,
+            cfg = rules_apple_transition_support.apple_rule_transition,
```

Comment:
Possible to add conditional assignment at the top of `framework.bzl` or in some other file? e.g. :
```
if rules_apple_api_version_3:
    apple_dep_transition = rules_apple_transition_support.apple_rule_transition
else:
    apple_dep_transition = apple_common.multi_arch_split
```

Then - consuming it here:
```
cfg = apple_dep_transition
```